README.md

```diff
@@ -199,13 +245,8 @@ DESC
   # ...  
 end
 ```
-
-Puppet Strings documents this way to preserve backwards compatibility with `puppet doc` and existing resource types.
-
-***Note: Puppet Strings does not evaluate your Ruby code, so only certain static expressions are supported.***
-
-To document parameters and properties that are dynamically created, use the `#@!puppet.type.param` and `#@!puppet.type.property`
-directives before the `newtype` call:
+ 
+If your resource type includes dynamically create parameters and properties, you must also use the `#@!puppet.type.param` and `#@!puppet.type.property` directives **before** the `newtype` call. This is necessary because Strings does not evaluate Ruby code, so it cannot detect dynamic attributes.
```

Comment:
Typo: "If your resource type includes dynamically create parameters and properties"